tests/plugins/font/manual/command/applystyle.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.14.0, feature, 3728
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, font, enterkey, elementspath, notification, sourcearea
+
+General Note:
+* Font and font size now use commands instead of direct operation on editor's content.
+* When font or fontSize command is executed then notification should be displayed in the editor and browser's console.
+* Please repeat steps for both font and font size features
+
+1. Select unstyled text and apply new font/fontSize.
+2. Try to apply this same font/fontSize again.
+3. Try to apply different font for already styled text.
```

Comment:
Looks like the following numbered list have been merged with a bullet list. It probably requires a double new line. Maybe also splitting them with additional heading would help?